snap_core22/snapcraft.yaml

```diff
@@ -0,0 +1,270 @@
+      - ros-humble-rclpy-message-converter-msgs
+      - ros-humble-rcss3d-agent-msgs
+      - ros-humble-realsense2-camera-msgs
+      - ros-humble-rmf-api-msgs
+      - ros-humble-rmf-building-map-msgs
+      - ros-humble-rmf-charger-msgs
+      - ros-humble-rmf-dispenser-msgs
+      - ros-humble-rmf-door-msgs
+      - ros-humble-rmf-fleet-msgs
+      - ros-humble-rmf-ingestor-msgs
+      - ros-humble-rmf-lift-msgs
+      - ros-humble-rmf-obstacle-msgs
+      - ros-humble-rmf-scheduler-msgs
+      - ros-humble-rmf-site-map-msgs
+      - ros-humble-rmf-task-msgs
+      - ros-humble-rmf-traffic-msgs
+      - ros-humble-rmf-visualization-msgs
+      - ros-humble-rmf-workcell-msgs
+      - ros-humble-robot-calibration-msgs
+      - ros-humble-robot-controllers-msgs
+      - ros-humble-rosapi-msgs
+      - ros-humble-rosbridge-msgs
+      - ros-humble-rosbridge-test-msgs
+      - ros-humble-rosgraph-msgs
+      - ros-humble-rtabmap-msgs
+      - ros-humble-rtcm-msgs
+      - ros-humble-rviz-2d-overlay-msgs
+      - ros-humble-sensor-msgs
+      - ros-humble-sensor-msgs-py
+      - ros-humble-shape-msgs
+      - ros-humble-soccer-object-msgs
+      - ros-humble-soccer-vision-2d-msgs
+      - ros-humble-soccer-vision-3d-msgs
+      - ros-humble-soccer-vision-attribute-msgs
+      - ros-humble-statistics-msgs
+      - ros-humble-std-msgs
+      - ros-humble-stereo-msgs
+      - ros-humble-stubborn-buddies-msgs
+      - ros-humble-system-modes-msgs
+      - ros-humble-teleop-tools-msgs
+      - ros-humble-test-msgs
+      - ros-humble-tf2-geometry-msgs
+      - ros-humble-tf2-msgs
+      - ros-humble-tf2-sensor-msgs
+      - ros-humble-trajectory-msgs
+      - ros-humble-turtlebot3-msgs
+      - ros-humble-turtlebot4-msgs
+      - ros-humble-tuw-geometry-msgs
+      - ros-humble-ublox-msgs
+      - ros-humble-ublox-ubx-msgs
+      - ros-humble-udp-msgs
+      - ros-humble-unique-identifier-msgs
+      - ros-humble-ur-dashboard-msgs
+      - ros-humble-ur-msgs
+      - ros-humble-urg-node-msgs
+      - ros-humble-velodyne-msgs
+      - ros-humble-vision-msgs
+      - ros-humble-visualization-msgs
+      - ros-humble-webots-ros2-msgs
+      - ros-humble-wiimote-msgs
+      - ros-humble-wireless-msgs
+#      - ros-humble-astuff-sensor-msgs  # new ppa, should we add it? https://github.com/astuff/pacmod3/commit/ff0f8f1bb1a1ba88f495a3294725207309c739ae
+#      - ros-humble-clearpath-msgs  # not yet released
+#      - ros-humble-clober-msgs  # not yet released
+#      - ros-humble-mocap-msgs  # not yet in humble, not sure why
+#      - ros-humble-pacmod3-msgs  # new ppa, should we add it? https://github.com/astuff/pacmod3/commit/ff0f8f1bb1a1ba88f495a3294725207309c739ae
+#      - ros-humble-raptor-dbw-msgs  # https://github.com/NewEagleRaptor/raptor-dbw-ros/commit/7feaf35d996addac469a0816ab0d7ab9037ca6dd
+#      - ros-humble-raptor-pdu-msgs  # https://github.com/NewEagleRaptor/raptor-dbw-ros/commit/7feaf35d996addac469a0816ab0d7ab9037ca6dd
```

Comment:
Same here, if they are not part of the official ROS 2 release it shouldn't be added otherwise everyone is going to add their custom msgs.